### PR TITLE
chore(RHTAPWATCH-823): SLO alerts uses team handles for routing

### PR DESCRIPTION
**Pull request by Omeramsc**
Now that SLO and non-SLO alerts are routed to different channels, there is no need to keep track of the namespace of each team. Instead, alert_routing_key is replaced by the slack's team handle, using the key:value `alert_team_handle`.

Since the team-handle for tagging in slack is using the team ID, a `team` key is also added to make the routing more readable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,22 +63,21 @@ where the team's handle is tagged in the alert message.
...
+Apply the `alert_team_handle` and `team` annotations to SLO alerts in order to get notified about them.
   
-#### How to apply the `alert_routing_key` Annotation:
+#### How to apply the `alert_team_handle` Annotation:
 
-Apply the `alert_routing_key` key to the annotations section of any alerting rule,
-with one of the team's namespaces as its value, or the team's name.
+Apply the `alert_team_handle` key to the annotations section of any alerting rule,
+with the relevant team's Slack handle.
+The format of the Slack handle is: `<!subteam^-team_id->` (e.g: `<!subteam^S041261DDEW>`).
```
Comment (yftacherzog) [inline README.md:72]: What's `team-id`? Is that the Slack group ID? We should use the same name used in Slack and maybe also explain how to retrieve it.

File: rhobs/alerting/data_plane/prometheus.latency_image_repository_provision_alerts.yaml [hunk 2]
```diff
@@ -6,29 +6,30 @@ metadata:
...
-      annotations:
-        summary: >-
-          Latency of image repository provision time exceeded
-        description: >
-          Time taken to provision image repository has been over
-          30s for more than 5% of requests during the last 5 minutes on cluster
-          {{ $labels.source_cluster }}
-        alert_routing_key: image-controller
-        runbook_url: https://gitlab.cee.redhat.com/rhtap/docs/sop/-/blob/main/image-controller/latency_image_repository_provision.md
+    - name: latency-image-repository-provision
```
Comment (yftacherzog) [inline rhobs/alerting/data_plane/prometheus.latency_image_repository_provision_alerts.yaml:9]: Is the fact that the entire block changed here is expected?

File: test/promql/tests/data_plane/release_service_test.yaml [hunk 3]
```diff
@@ -6,14 +6,22 @@ evaluation_interval: 1m
 tests:
   - interval: 1m
     input_series:
-      - series: 'release_processing_duration_seconds_bucket{le="3600", job="release", namespace="foo", source_cluster="cluster01"}'
```
Comment (yftacherzog) [inline test/promql/tests/data_plane/release_service_test.yaml:9]: Is this change related to this PR?

**Review by yftacherzog — COMMENTED** [review]
There are a few changes that looks like fixing of indentations etc. I would prefer this not to be done as part of such a big PR. There's enough potential for things to break as-is. Not sure we need to add additional changes along the way.

File: test/promql/tests/data_plane/release_service_test.yaml (hunk 3 above)
Comment (Omeramsc) [reply test/promql/tests/data_plane/release_service_test.yaml:9]: fixed

**Review by yftacherzog — APPROVED** [review]
lgtm

**Review by amisstea — APPROVED** [review]
